contributing/deployment-tools.txt

```diff
@@ -0,0 +1,153 @@
+OME deployment tools
+====================
+
+This section describes deployment tools supported by the OME team.
+It is primarily designed for the core OME developers who want to
+bring new or upgrade existing prerequisites. The following steps explain
+the connections between basic repositories and the testing workflow.
+
+
+.. note::
+
+    This section requires a brief understanding of Ansible 
+    http://docs.ansible.com/ansible/intro_getting_started.html
+    and Docker engine https://docs.docker.com/.
+
+Prerequisites locations
+-----------------------
+
+List of OME prerequisites is stored in multiple git repositories, each of 
+which is available from several locations.
+
+OME Infrastructure
+^^^^^^^^^^^^^^^^^^
+
+Infrastructure is provided to simplify deployment using 
+`Ansible <https://www.ansible.com>`_.
+
+The Infrastructure repository is available from:
+
+-  https://github.com/openmicroscopy/infrastructure
+
+Roles repositories are available in:
+
+-  https://github.com/openmicroscopy?q=ansible-role
+
+
+OMERO-install
+^^^^^^^^^^^^^
+
+OMERO installation scripts are provided to help new users with installing
+OMERO.server for the first time on a clean system, and can be used as
+the basis for more advanced configurations.
+
+The OMERO-install repository is available from:
+
+-  https://github.com/ome/omero-install
+
+The OMEROWEB-install repository is available from:
+
+-  https://github.com/ome/omeroweb-install
+
+Testing workflow
+----------------
+
+Testing environment is split into two tiers: developer and production
+testing platform.
+
+
+Devspace - Continuous Integration (Dev Testing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Continuous integration tools managed by Jenkins CI providing
+an automation framework that runs repeated jobs. The default deployment
+initializes a Jenkins CI master with a predefined set of jobs.
+
+The :doc:`Devspace <devspace>` repository is available from:
+
+-  https://github.com/openmicroscopy/devspace
+
+Devspace Dockerfiles uses common devslave image. The Devslave repository
+is available from:
+
+-  https://github.com/openmicroscopy/devslave-c7-docker
+
+
+CI-master - Continuous Delivery (Production)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Production Continuous Delivery (CD) platform managed by
+`Jenkins <http://jenkins-ci.org>`_ available on http://ci.openmicroscopy.org.
+More details about CI-master available on :doc:`continuous-integration`
+
+
+How to add new/upgrade/remove old prerequisites
+-----------------------------------------------
+
+When the OME platform requires a new set of prerequisites all the above
+listed repositories may require updates. Depends on the nature of 
+packages developers must consider:
+
+-  infrastructure repository:
+
+     - adding new Ansible role in its own repo or update existing one
```

Comment:
All of these sub-bulletpoints have purple highlighting on https://www.openmicroscopy.org/site/support/contributing-staging/deployment-tools.html - you could try indenting them one space less and seeing if they still nest?